src/Domain/TicTacToeGame.cs

```diff
@@ -0,0 +1,202 @@
+using System;
```

Comment:
@maracuja-juice Please review. The logic should be implemented. I will write tests soon to validate.

Reply:
This looks great! I'm confident that it works correctly but yes, add some tests.

Reply:
Tests show that it is somehow not working yet

Reply:
The error seems to be fixed by implementing Equals on Position

Reply:
:confetti_ball: